packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr-add-step.ts

```diff
@@ -0,0 +1,99 @@
+import iam = require('@aws-cdk/aws-iam');
+import sfn = require('@aws-cdk/aws-stepfunctions');
+import { Stack } from '@aws-cdk/core';
+import { getResourceArn } from './resource-arn-suffix';
+
+/**
+ * Properties for EmrAddStep
```

Comment:
Since this set of classes does not use L2s to interface with the outside world, and I don't want to block it for too long, can you please add `@experimental` to the docstring of every class, interface and enum?

That will give us the opportunity to change it in the future.

Reply:
all classes, interfaces, and enums marked experimental